docs/benchmarks/performance_benchmarks_openvino.md

```diff
@@ -6,24 +6,44 @@
    :hidden:
 
    openvino_docs_performance_benchmarks_faq
-   Download Performance Data Spreadsheet in MS Excel* Format <https://docs.openvino.ai/downloads/benchmark_files/OV-2022.1-Download-Excel.xlsx>
+   Download Performance Data Spreadsheet in MS Excel Format <https://docs.openvino.ai/downloads/benchmark_files/OV-2022.1-Download-Excel.xlsx>
    openvino_docs_performance_int8_vs_fp32
 
 
 @endsphinxdirective
 
-This benchmark setup includes a single machine on which both the benchmark application and the OpenVINO™ installation reside.
+Features and benefits of Intel® technologies depend on system configuration and may require enabled hardware, software or service activation. You can learn more on this subject from the original equipment manufacturer (OEM), official [Intel® web page](https://www.intel.com) or retailer.
 
-The benchmark application loads the OpenVINO™ Runtime and executes inferences on the specified hardware (CPU, GPU or VPU). The benchmark application measures the time spent on actual inferencing (excluding any pre or post processing) and then reports on the inferences per second (or Frames Per Second). For more information on the benchmark application, please also refer to the entry 5 of the [FAQ section](performance_benchmarks_faq.md).
+## Platform Configurations
+
+@sphinxdirective
+
+:download:`A full list of used for testing HW platforms (along with their configuration)<../../../docs/benchmarks/files/Platform_list.pdf>`
+
+@endsphinxdirective
+
+For more detailed configuration descriptions, see the [Configuration Details](https://docs.openvino.ai/resources/benchmark_files/system_configurations_2022.1.html) document.
+
+## Benchmark Setup Information
+
+This benchmark setup includes a single machine on which both the benchmark application and the OpenVINO™ installation reside. The presented performance benchmark numbers are based on realease 2022.1 of Intel® Distribution of OpenVINO™ toolkit.
+
+The benchmark application loads the OpenVINO™ Runtime and executes inferences on the specified hardware (CPU, GPU or VPU). It measures the time spent on actual inferencing (excluding any pre or post processing) and then reports on the inferences per second (or Frames Per Second - FPS). For additional information on the benchmark application, refer to the entry 5 in the [FAQ section](performance_benchmarks_faq.md).
 
-Measuring inference performance involves many variables and is extremely use-case and application dependent. We use the below four parameters for measurements, which are key elements to consider for a successful deep learning inference application:
+Measuring inference performance involves many variables and is extremely use case and application dependent. Below four parameters for measurements our used for measurrment. They are key elements to consider for a successful deep learning inference application:
```

Comment:
```suggestion
Measuring inference performance involves many variables and is extremely use case and application dependent. Below are four parameters used for measurements, which are key elements to consider for a successful deep learning inference application:
```

I guess there was too much of ambiguity, whether the parameters or measurements are key elements. 
Still, it might be just me ;)